packages/wallet/package.json

```diff
@@ -68,7 +68,7 @@
     "object-hash": "^2.0.0",
     "promise": "8.0.1",
     "prop-types": "^15.6.2",
-    "pure-evm": "^0.1.2",
+    "pure-evm": "0.1.3",
```

Comment:
This was the change, by the way: https://github.com/armaniferrante/pure-evm/commit/c316a802e3305bb6f38557918bdf4da32e2433bc